ContriHub/settings.py

```diff
@@ -65,15 +66,15 @@
 
 
 INSTALLED_APPS = [
+    'Users',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-
     'Projects',
-    'Users',
```

Comment:
why did you remove Users from installed apps

Reply:
Sir actually I moved Users to the top in order to use the created forgot password template. When it was at the bottom forgot password was actually using the custom django template for forgot password